ScoutSuite/output/result_encoder.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import datetime
+import dateutil
+import json
+import os
+from sqlitedict import SqliteDict
+
+from ScoutSuite.core.console import print_exception, print_info
+
+from ScoutSuite import DEFAULT_REPORT_DIR
+from ScoutSuite.output.utils import get_filename, prompt_for_overwrite
+
+
+class ScoutJsonEncoder(json.JSONEncoder):
+    """
+    JSON encoder class
+    """
+
+    def default(self, o):
+        try:
+            if type(o) == datetime.datetime:
+                return str(o)
+            else:
+                # remove unwanted attributes from the provider object during conversion to json
```

Comment:
I think we'll be able to delete that when #183 will be fully merged, since those won't exist anymore.